src/com/owncloud/android/datamodel/OCFile.java

```diff
@@ -60,6 +60,47 @@ public OCFile createFromParcel(Parcel source) {
     public static final String PATH_SEPARATOR = "/";
     public static final String ROOT_PATH = PATH_SEPARATOR;
 
+    public enum AvailableOfflineStatus {
+
+        /**
+         * File is not available offline
+         */
+        NOT_AVAILABLE_OFFLINE(0),
+
+        /**
+         * File is available offline
+         */
+        AVAILABLE_OFFLINE(1),
+
+        /**
+         * File belongs to an available offline folder
+         */
+        AVAILABLE_OFFLINE_PARENT(2);
+
+        private final int value;
+
+        AvailableOfflineStatus(int value) {
+            this.value = value;
+        }
+
+        public int getValue() {
+            return value;
+        }
+
+        public static AvailableOfflineStatus fromValue(int value) {
```

Comment:
you could use existing .ordinal() method and values()[index] to get them so that you wouldn't need to have a value attribute (though maybe less control as you'd need to take care to add new enum values at the end just to preserve existing indexes)